docs/command-line-interface/index.md

```diff
@@ -1,231 +0,0 @@
----
```

Comment:
I think this probably needs to live in the eslint repo, since if we make changes to the CLI, we'll want this page updated with the PR to reflect that change.